tests/gdx-tests/src/com/badlogic/gdx/tests/gwt/GwtTestWrapper.java

```diff
@@ -523,6 +525,10 @@ public GdxTest instance () {
 			}
 		}, new Instancer() {
 			public GdxTest instance () {
+				return new FramebufferToTextureTest();
```

Comment:
I expected that it has been excluded due to the lack of the ScreenUtils. But there seem to be some Web GL problems.